zingo-netutils/src/lib.rs

```diff
@@ -16,7 +16,8 @@ use tonic::Status;
 use tower::util::BoxCloneService;
 use zcash_client_backend::proto::service::compact_tx_streamer_client::CompactTxStreamerClient;
 
-type UnderlyingService = BoxCloneService<
+/// TODO: add doc-comment
```

Comment:
Why is this type being defined?

Reply:
well, you may have a better idea here but I think it's a lot to define without this type alias. any suggestions welcome

Reply:
Please replace the TODO: with something else.

Reply:
Yeah sure

Reply:
Oh I see it's just being made `pub`...